docs/UserGuide.md

```diff
@@ -139,6 +139,20 @@ Examples:
 * `unenrol m/1 e/5` unlinks the 1st member in the member list from the 5th event in the event list.
 * `unenrol m/5 e/1` unlinks the 5th member in the member list from the 1st event in the event list.
 
+### Edit an enrolment: editAttendance
+Edits the attendance details of a specified member at a specified event with the specified attributes.
+
+Format: `editAttendance m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]`
+
+* Links the member at the specified `MEMBER_INDEX` to the event at the specified `EVENT_INDEX` with `NUMBER_OF_HOURS` specifying the number of hours that the member contributed and `REMARK` stating extra remarks about the member and event.
```

Comment:
Should this be edits the linkage or something to that meaning?

```suggestion
* Edits the linkage between the member at the specified `MEMBER_INDEX` and the event at the specified `EVENT_INDEX` with `NUMBER_OF_HOURS` specifying the number of hours that the member contributed and `REMARK` stating extra remarks about the member and event.
```

Reply:
Maybe like this?
```suggestion
* Edits the specified `MEMBER_INDEX`'s attendance of the event at the specified `EVENT_INDEX` with `NUMBER_OF_HOURS` specifying the number of hours that the member contributed and `REMARK` stating extra remarks about the member and event.
```

Reply:
Agreed, will make the change.